metriq_gym/cli.py

```diff
@@ -2,6 +2,47 @@
 
 import argparse
 
+from tabulate import tabulate
+
+from metriq_gym.job_manager import JobManager, MetriqGymJob
+from metriq_gym.provider import ProviderType
+
+LIST_JOBS_HEADERS = ["ID", "Provider", "Device", "Type", "Dispatch time (UTC)"]
+TIMESTAMP_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+
+def list_jobs(args: argparse.Namespace, job_manager: JobManager) -> int:
+    """List jobs recorded in the job manager.
+
+    Args:
+        args: Parsed arguments.
+        job_manager: Job manager instance.
+    Returns:
+        Return code.
```

Comment:
It seems weird that this opts for a return code. I think this should likely return `None`. The only code that is returned is `0`. So I think we can: 

- Remove the `return 0` statements
- Change the return type from `int` to `None`
- Update the `Returns` section in the docstring (remove it).